src/main/java/billgates/view/gui/MainFrame.java

```diff
@@ -5,7 +5,7 @@
 import javax.swing.border.LineBorder;
 import java.awt.*;
 
-public class MainFrame extends JFrame {
+public class MainFrame extends JFrame{
```

Comment:
I think there should be a space before `{`

Reply:
Fix it